extensions-support/ahc/deployment/pom.xml

```diff
@@ -42,6 +42,11 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-support-ahc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.graalvm.nativeimage</groupId>
+            <artifactId>svm</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
I guess this should rather go to the runtime module, where we do substitutions?

Possible followup: we should slowly Migrate from `org.graalvm.nativeimage:svm` to `org.graalvm.sdk:graal-sdk` https://github.com/apache/camel-quarkus/issues/4602

Reply:
I see, I added dependency to the wrong module. I'll fix it in a few minutes.

Reply:
fixed